src/components/JobBuildsHistogram.jsx

```diff
@@ -35,7 +35,7 @@ class JobBuildsHistogram extends Component {
         const data = builds.map(build => ({
             x:      build.number,
             y:      build.duration / 1000 / 60, // converts ms to mn
-            result: build.result ? build.result.toLowerCase() : 'unkown'
+            result: build.result ? build.result.toLowerCase() : 'unknown'
```

Comment:
Oh, if you mean from util.js...that is because `build.result` will be undefined so it won't hit the .toLowerCase() method that causes the failure.